resume.md

```diff
@@ -1,49 +1,52 @@
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Languages**: Python, Java, C++, SQL, Javascript, HTML, CSS
+* **Libraries/Frameworks**: React, TensorFlow, PyTorch
+* **Version Control**: Git, GithubDesktop
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Project Management Intern**: Intensivate | Berkeley, CA ... *Sep. 2020 – Present*
+* Led international team of 6 to develop an internal framework host server to power real-time collaboration software using JS & Nginx
+* Facilitated biweekly standup meetings, established communication between team & client contact, and defined/tracked deliverables
+* Relayed user testing feedback on Propend tool to facilitate rapid production of technical specifications for novel server architecture
 
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
+**Contract Consultant**: SoFi | Berkeley, CA ... *Sep. 2020 – Present*
+* Conducted primary & secondary market research in p-finance and POS verticals to relay market trends and demand for product
+* Analyzed multiple datasets from targeted campaigns using Excel to identify market trends and gauge relative value of unique features
+* Compiled & presented 60+ PowerPoint slide deck containing strategy, marketing & product decisions to SoFi upper management
 
 ## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
+**stereoAlgo**: *Developed stereoscopic algorithm to construct accurate disparity map 4 times faster than industry leader* | *Aug 2018 – Mar 2019*
+* Developed a stereoscopic optical detection algorithm to autonomously construct and respond to an accurate real-time disparity map of foreign environments using a self-developed lightweight computer vision algorithm
```

Comment:
Interesting - this sounds technical and speaks more to your software engineering experience.

What is the outcome of this work?  Yes you used computer vision but in layman terms what happened?